p4-16/spec/P4-16-spec.mdk

```diff
@@ -6240,7 +6242,7 @@ table smac {
 ~ End P4Example
 
 - The entries in the `smac` `table` may contain two different actions: `Drop_action` and `Rewrite_mac`.
-- The `Rewrite_smac` action has one parameter, `sourceMac`, which is bound by the control plane.
+- The `Rewrite_smac` action has one parameter, `sourceMac`, which will be provided in the table entry.
```

Comment:
Is this change in the wrong place? For this example the data is bound by the control-plane.

Reply:
Fixed. Thanks.